pkg/build/build.go

```diff
@@ -1690,6 +1699,46 @@ func (ctx *Context) BuildPackage() error {
 		return err
 	}
 
+	// TODO: Should this go here?
```

Comment:
I think this should be factored out to a new function, perhaps `WorkspaceCopyBack` or similar.